doc/contributing/releases.md

```diff
@@ -721,6 +721,23 @@ Revert all changes that were made to `src/node_version.h`:
 $ git checkout --ours HEAD -- src/node_version.h
 ```
 
+<details>
+<summary>LTS</summary>
```

Comment:
I think this should be 'Major version' rather than LTS

Reply:
I'll go with "Major version release" to fit well with the already existing "Security release" details / summary sections

Reply:
Commit title will also need to be updated.